layouts/index.html

```diff
@@ -230,7 +230,13 @@ <h3>{{ .Params.video.heading }}</h3>
 
         <!-- Rendered with Hugo's YouTube shortcode: https://gohugo.io/content-management/shortcodes/#youtube -->
         <div class="rounded shadow-md" style="position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden;">
-            <iframe src="//www.youtube.com/embed/6f8KF6UGN7g?rel=0" style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; border:0;" allowfullscreen="" title="Introduction to Pulumi: Modern Infrastructure as Code"></iframe>
+            <iframe
+                src="//www.youtube.com/embed/6f8KF6UGN7g?rel=0"
+                style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; border:0;"
+                allowfullscreen=""
+                title="Introduction to Pulumi: Modern Infrastructure as Code"
+                srcdoc="<style>*{padding:0;margin:0;overflow:hidden}html,body{height:100%}img,span{position:absolute;width:100%;top:0;bottom:0;margin:auto}span{height:1.5em;text-align:center;font:48px/1.5 sans-serif;color:white;text-shadow:0 0 0.5em black}</style><a href=https://www.youtube.com/embed/6f8KF6UGN7g?autoplay=1><img src=https://img.youtube.com/vi/6f8KF6UGN7g/hqdefault.jpg alt='Introduction to Pulumi: Modern Infrastructure as Code'><span>▶</span></a>"
```

Comment:
@cnunciato What do you think of this? We should get a cleaner image for the preview if we go this route, but it makes a huge difference in page speed. Once I did all the image optimizations we were getting ~48 for the page speed and once I added this `srcdoc` attribute we got up to ~70.